sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/algorithms/aes_cbc.py

```diff
@@ -12,60 +10,80 @@
 from ..transform import BlockCryptoTransform
 
 
+_CBC_BLOCK_SIZE = 128
+
+
 class _AesCbcCryptoTransform(BlockCryptoTransform):
     def __init__(self, key, iv):
         super(_AesCbcCryptoTransform, self).__init__(key)
         self._cipher = Cipher(algorithms.AES(key), modes.CBC(iv), backend=default_backend())
 
     def transform(self, data):
-        return self.update(data) + self.finalize(data)
-
-    @abstractmethod
+        return self.update(data) + self.finalize()
+    
     def block_size(self):
-        pass
+        return _CBC_BLOCK_SIZE
 
 
 class _AesCbcDecryptor(_AesCbcCryptoTransform):
     def __init__(self, key, iv):
         super(_AesCbcDecryptor, self).__init__(key, iv)
         self._ctx = self._cipher.decryptor()
-        self._padder = padding.PKCS7(self.block_size).unpadder()
+        self._padder = padding.ANSIX923(_CBC_BLOCK_SIZE).unpadder()  # TODO: Update this to zero-padding
```

Comment:
If the current code doesn't do the right thing, and this change doesn't fix that, I wouldn't make the change